packages/blob-reader/fixtures/github.com/issue/code.html

```diff
@@ -18,48 +17,65 @@
 
 
 
-  <link crossorigin="anonymous" media="all" integrity="sha512-InGiw0uqrfrV1EXLEoQR8zhoNQs4RhOXAZIoTEr+QaNPzbnLcqgPASwPZOSxhLC/ydy1tqek+nufQayviU4NRA==" rel="stylesheet" href="https://github.githubassets.com/assets/frameworks-c0ad6862340d6d1a2594b40111c17bf7.css" />
-  <link crossorigin="anonymous" media="all" integrity="sha512-q0uE9UAtjRgctFaLFVvUGVtXC6ggCD9Qcg86uwRaLx2RdC7FIE+hPfB4YDaM8lCecPVMwsxDyvqkXI2XEKDCOA==" rel="stylesheet" href="https://github.githubassets.com/assets/site-d2ea9f95eb7121ca3a5f9571e0012f11.css" />
-    <link crossorigin="anonymous" media="all" integrity="sha512-JedmoBWaUotKU3Fc6fDpBbcfTTMhoAiSb4IPTqwmqhD3pBHWUtHDyYYgVGuNwIKt08sVud5nLKZCb8DBdPvtNQ==" rel="stylesheet" href="https://github.githubassets.com/assets/github-0c16774092b1ce5288b91585e3d6617b.css" />
+  <link crossorigin="anonymous" media="all" integrity="sha512-QLgu+HpQvHV00FunhsB3ueQCfgM7dkyMXKp1BrwCifVinFow2alu90jbiIbSdyn4/rXsaRo3otCqeYgmRmTCEA==" rel="stylesheet" href="https://github.githubassets.com/assets/frameworks-40b82ef87a50bc7574d05ba786c077b9.css" />
```

Comment:
All changes made in this file are due to the fact that I added a new snipped to https://github.com/OctoLinker/testrepo/issues/2. I ran `./packages/blob-reader/scripts/update-fixtures.sh` and committed only this file to keep the overall diff of this PR small. Interesting to see all the markup changes

Reply:
Yes, I'm so glad that OctoLinker is not tight to the DOM anymore. At the beginning OctoLinker stop working every now and then. The regular expression approach and https://github.com/padolsey/findAndReplaceDOMText is the key why OctoLinker is stable since many years